corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
then the breakdown would be:

- `settings.py' is 100% static/shared
- all custom variables are accessed through `settings_manager`
- all `settings_manager` custom variables get set through one or more static config files

(note that I'm just thinking out loud, not saying this is obviously better than other approaches)